### PR TITLE
DCOS-10991 [4 of 4]: Volumes form section

### PR DESCRIPTION
**Pull request by Poltergeist**
volumes mov

This adds the form component to add volumes.

### CODE REVIEW

File: src/js/utils/__tests__/ReducerUtil-test.js [hunk 1]
```diff
@@ -86,6 +86,62 @@ describe('ReducerUtil', function () {
         .toEqual([{id: 'foo', vip: 'foo:8080'}, {id: 'bar', vip: undefined}]);
     });
 
+    it('should use context in nested combineReducers', function () {
+      const reducers = ReducerUtil.combineReducers({
+        id: idReducer,
+        container: ReducerUtil.combineReducers({
+          vip(state = undefined, action) {
```
Comment (mlunoe) [inline src/js/utils/__tests__/ReducerUtil-test.js]: the default `state` assignment to `undefined` has no effect

File: plugins/services/src/js/reducers/JSONConfigReducers.js [hunk 2]
```diff
@@ -11,7 +12,15 @@ import {
 module.exports = {
   id: simpleReducer('id'),
   instances: simpleIntReducer('instances'),
-  container,
+  container: combineReducers({
```
Comment (mlunoe) [inline plugins/services/src/js/reducers/JSONConfigReducers.js]: We probably need to re-consider this for when we introduce the multiple-container, but this is a good stepping stone

File: src/js/utils/__tests__/ReducerUtil-test.js (hunk 1 above)
Comment (Poltergeist) [inline src/js/utils/__tests__/ReducerUtil-test.js]: Addressed this issue in all cases.

File: plugins/services/src/js/reducers/JSONConfigReducers.js (hunk 2 above)
Comment (mlunoe) [inline plugins/services/src/js/reducers/JSONConfigReducers.js]: Ignore this, as it is handled by switching reducers!

**Comment by mlunoe** [comment]
Ok, going to merge this in. I added a parser to make sure we are parsing integers for sizes.

A few follow-ups:
- We are missing the option to add a "Host" container, which in the old create form was referred to as "Docker Container Volumes". See image:

- External volumes can take size, but let's handle that in a separate issue (created a JIRA), which we can prioritise and add after the release.
- Since there is only one selection for external volumes, let's remove the select box, with only one option

**Pull request merged**